tracer-transformers/example2/Main.hs

```diff
@@ -70,8 +72,10 @@ condVerbosity v = condTracingM (const <$> verbosityP v)
 -- The distinct message types that you want to expose. This is the set
 -- of observables.
 
-data MyMessages a = MyStart a | MyNormal a | MyWarning (Int, a)
-                  deriving Show
+data MyMessages a = MyStart a
+    | MyNormal a
+    | MyWarning (Int, a)
+    deriving Show
```

Comment:
Similrly, the ADT variants are formatted oddly.. as well as in the remaining cases.